src/agoradatatools/etl/transform/modelAD_general_transform.py

```diff
@@ -0,0 +1,52 @@
+"""
+This module contains the transformation logic for the biomarkers and pathology datasets.
+This is for the Model AD project.
+"""
+
+import pandas as pd
+from typing import Dict, List
+
+
+def modelAD_general_transform(
```

Comment:
Is this transform going to be useful for more datasets than just these two? I am wondering if a more specific name that says this is for biomarkers and pathology would be more appropriate